### ISSUE TITLE
Exiting without showing any error!

### ISSUE THREAD

**Issue by agrawalsmart7** [AlephNullSK/dnsgen#2]
Hi, I have installed all the modules correctly but still, it didn't run. It exits silently without showing any error.

I run this command `$ python3 dnsgen.py`
And it exits back to my directory?

**Comment by PatrikHudak** [comment]
You need to run `python3 setup.py install` in order to install CLI. `dnsgen.py` is a module that does not run any code on start.

**Comment by agrawalsmart7** [comment]
Yeah, I already did, but still, that happening now I have no idea what is going wrong?.

**Comment by agrawalsmart7** [comment]
Now it's working, one thing I want to suggest you is to add the DNS resolver. Yeah, we can use massdns but if it's by default in your tool it will be more good. But anyway the tool is really good.
